build-tools/installers/create-installers.targets

```diff
@@ -386,6 +386,12 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Debugging.Tasks.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Common.Debugging.props" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Common.Debugging.targets" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\arm64-v8a\installer" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\armeabi-v7a\installer" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\x86\installer" />
```

Comment:
@BretJohnson I think there is an existing `lib` folder where we put all the native android libraries. Perhaps we should put these tools in there too? I was panning to put the native fastdev 2 stuff in the `lib/xxxx` folders.

Reply:
@dellis1972 I moved the installers to the lib directory, with this PR: https://github.com/xamarin/androidtools/pull/231
Can you please review & approve. With that, the `installer` executable will be in the e.g. `lib/x86` subdirectory, next to the other device specific binaries.

Reply:
@BretJohnson That PR was merged can you update this PR with the new paths :)

Reply:
Thanks. And the paths are now updated.